editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.spec.browser.tsx

```diff
@@ -479,28 +526,33 @@ describe('Spy Wrapper Template Path Tests', () => {
         ":storyboard": Object {
           "children": Array [],
           "name": "Storyboard",
+          "rootElements": Array [],
         },
         ":storyboard/scene": Object {
-          "children": Array [
+          "children": Array [],
+          "name": "div",
+          "rootElements": Array [
             "storyboard/scene:app-root",
           ],
-          "name": "div",
         },
         "storyboard/scene:app-root": Object {
           "children": Array [
             "storyboard/scene:app-root/inner-div",
           ],
           "name": "div",
+          "rootElements": Array [],
         },
         "storyboard/scene:app-root/inner-div": Object {
           "children": Array [
             "storyboard/scene:app-root/inner-div/card-instance",
           ],
           "name": "div",
+          "rootElements": Array [],
         },
         "storyboard/scene:app-root/inner-div/card-instance": Object {
           "children": Array [],
           "name": "div",
+          "rootElements": Array [],
```

Comment:
hmm, shouldn't this list `"storyboard/scene:app-root/inner-div/card-instance:button-instance"` as a rootElement?

Reply:
It should eventually yes, but right now the dom walker isn't capturing that information because of the different paths for scenes vs elements, which this work will address.

Reply:
Oh which maybe means that we can't replace the `getAllChildrenIncludingUnfurledFocusedComponents` function just yet

Reply:
yeah